examples/AIcargo/model_hosting_1/settings.py

```diff
@@ -217,3 +217,11 @@
         },
     },
 }
+
+FILE_SYSTEM = os.environ.get('FILE_SYSTEM', 'local')
+FILE_SYSTEM_PREFIX = os.environ.get('FILE_SYSTEM_PREFIX', './')
+UPLOAD_PATH = os.environ.get('UPLOAD_PATH', 'uploaded')
+MODEL_PATH_PREFIX = os.environ.get('MODEL_PATH_PREFIX', 'model_repo')
```

Comment:
I think it would be good to add the prefix `TF_` to the above constants as they are only relevant for TensorFlow.